Bottom: f2bbd90e42ffecc5526a8a89eee1ea85006ba567
Top:    9e5ede7791f6b2511004117b075778269baa6fc8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-15 16:10:24 +0000

Refresh of drop-in-pcb-gl-code-various-me

---

diff --git a/src/crosshair.c b/src/crosshair.c
index 601d5d6..cea493a 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -44,7 +44,6 @@
 #include "data.h"
 #include "draw.h"
 #include "error.h"
-
 #include "line.h"
 #include "misc.h"
 #include "mymem.h"
